Hearthstone Deck Tracker/LogReader/Handlers/TagChangeActions.cs

```diff
@@ -65,10 +65,27 @@ public Action FindAction(GameTag tag, IGame game, IHsGameState gameState, int id
 					return () => OnCardCopy(id, value, game, gameState);
 				case TAG_SCRIPT_DATA_NUM_1:
 					return () => OnTagScriptDataNum1(id, value, game, gameState);
+				case REBORN:
+					return () => RebornChange(id, value, game);
 			}
 			return null;
 		}
 
+		private void RebornChange(int id, int value, IGame game)
+		{
+			if(value != 1)
```

Comment:
This should check that the game mode is battlegrounds. See e.g. `OnTagScriptDataNum1` below.